playbooks/loki.yaml

```diff
@@ -0,0 +1,44 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Configure loki.
+---
+- name: Configure loki
+  hosts: loki_servers
+  become: true
+  tasks:
+    - name: Install loki
+      ansible.builtin.package:
+        name: loki
+        state: present
+
+    - name: Configure loki
+      ansible.builtin.template:
+        src: files/loki/config.yml.j2
+        dest: /etc/loki/config.yml
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Install ferm configuration
+      ansible.builtin.template:
+        src: files/loki/ferm.conf.j2
+        dest: /etc/ferm/ferm.d/svc-loki-{{ item }}.conf
+        owner: root
+        group: adm
+        mode: '0644'
+      loop:
+        - http
+        - grpc
+
+    - name: Start and enable service
+      ansible.builtin.service:
+        name: loki
+        state: started
+        enabled: true
+      notify: _tina_reload_ferm
```

Comment:
I think this got mixed up, there is no need to reload ferm when you start or enable loki, only when the ferm config changes.
```suggestion
```